src/openforms/formio/rendering/default.py

```diff
@@ -345,3 +350,60 @@ def is_visible(self) -> bool:
         registration data...
         """
         return False
+
+
+@register("partners")
+class PartnersNode(ComponentNode):
+    @property
+    def value(self) -> Any:
+        value = glom(self.step_data.data, self.path or self.key)
+        return value
+
+    @property
+    def display_value(self) -> str | Any:
+        # in export mode, expose the raw datatype
+        if self.mode == RenderModes.export:
+            return self.value
+
+        if isinstance(self.value, list):
+            return ""
+
+        return format_value(self.component, self.value, as_html=self.renderer.as_html)
```

Comment:
I think we have some kitchensink tests somewhere that should lead to this codepath being covered too